src/Orckestra.Composer.Website/UI.Package/Typescript/Controller/AutocompleteSearchBoxVueController.ts

```diff
@@ -174,12 +176,17 @@ module Orckestra.Composer {
                         this.searchMore();
                     },
                     selectedCategorySuggestion(suggestion) {
-                        EventHub.instance().publish('categorySuggestionClicked', {
-                            data: {
-                                suggestion: suggestion.item.DisplayName,
-                                parents: suggestion.item.Parents
-                            }
-                        });
+                        if (this.sectionConfigs.suggestcategories.displayCategoryPage) {
+                            window.location.href = suggestion.item.Url;
+                        }
+                        else {
+                            EventHub.instance().publish('categorySuggestionClicked', {
+                                data: {
+                                    suggestion: suggestion.item.DisplayName,
+                                    parents: suggestion.item.Parents
+                                }
```

Comment:
we need to publish event and Analitics subscribed to it.
Move the logic inside 
categorySuggestionClicked(eventInformation: Orckestra.Composer.IEventInformation)